extra-devel/capstone/autobuild/beyond

```diff
@@ -0,0 +1,5 @@
+abinfo "Making capstone Python 3 bindings ..."
+cd "$SRCDIR"/bindings/python
```

Comment:
Consider using functions exported in `build/15-python.sh`.

Reply:
Python 3 binding here can be considered as a standalone project, I think it may be the reason why build with absolute path will end up with failure.